ensemble/ctf/utils.py

```diff
@@ -1,7 +1,10 @@
 from __future__ import unicode_literals
+
+from io import open
```

Comment:
Why not use the builtin `open`?

Reply:
Ahh. When I was initially debugging the issue with saving JSON data, I wanted to open the file with `encoding` kwarg, which isn't an option with the builtin `open` on Python 2. We don't need this anymore. Reverting to using the builtin `open`.